dali/test/python/sequences_test_utils.py

```diff
@@ -28,6 +28,49 @@
 vid_file = os.path.join(data_root, 'db', 'video',
                         'sintel', 'sintel_trailer-720p.mp4')
 
+class ParamsProvider:
+    def __init__(self, input_params):
```

Comment:
How about:
```suggestion
    def __init__(self, **input_params : ParamDesc):
```
?

Reply:
I wanted to leave existing tests as they are. I split the ParamsProvider into a base class and actual class, the first is just about providing input data to the provider instance while the actual one take input arguments description (added docs there with some type annotation) and computes parameters. This way derived classes can easily control how to describe tensor arguments.